code/game/objects/items/weapons/implants/implant_misc.dm

```diff
@@ -48,6 +48,24 @@
 	imp_in.reagents.add_reagent("omnizine", 10)
 	imp_in.reagents.add_reagent("stimulants", 10)
 
+/obj/item/weapon/implant/adrenalin/deathsquad
+	name = "adrenal implant"
+	desc = "Removes all stuns and knockdowns."
+	icon_state = "adrenal"
+	origin_tech = "bluespace=4;biotech=10;combat=10;syndicate=4"
+	uses = 300
```

Comment:
@Shadowlight213 I don't think there's ever going to be a time where it actually gets used more than 50x in a round considering you OD on omnizine and synaptizine pretty quick. Maybe 200 uses got burned up in training.